ingestion/functions/parsing/ch_zurich/input_event.json

```diff
@@ -1,10 +1,6 @@
 {
     "s3Bucket": "epid-sources-raw",
-    "s3Key": "5f32be3fe3f3ef002e849307/2020/08/11/1552/content.csv",
+    "s3Key": "5f32be3fe3f3ef002e849307/2020/08/12/0815/content.csv",
     "sourceUrl": "https://github.com/openZH/covid_19/blob/master/fallzahlen_kanton_alter_geschlecht_csv/COVID19_Fallzahlen_Kanton_ZH_alter_geschlecht.csv",
-    "sourceId": "5f32be3fe3f3ef002e849307",
-    "dateFilter": {
```

Comment:
Was the dateFilter intentionally dropped?

Reply:
Was just testing, no big deal if it's dropped or not.